examples/deployment/kubernetes/README.md

```diff
@@ -50,12 +50,39 @@ curl -X POST ${LOG_URL}/v1beta1/logs/${tree_id}:init
 
 ```
 
-The easiest way to do this is probably to use `kubectl exec <name of one of the logserver pods> -ti -- /bin/bash` to get a shell on a logserver Pod, and use curl from there.
+The easiest way to do this is to port-forward one of the logserver pods to your local workstation `kubectl port-forward <name of one of the logserver pods> 8091:8091` and use curl to access the API endpoint:
 
-(Use `kubectl get pods` to retrieve a list of all the Pods.)
+1. Grab the name of 0th logserver pod:
+```
+LOG_POD=$(\
+  kubectl get pods \
+  --selector=io.kompose.service=trillian-log \
+  --output=jsonpath='{.items[0].metadata.name}')
+```
+1. Port forward this pod's port `8091` to your local workstation (we're using `8091` locally too):
+```
+kubectl port-forward ${LOG_POD} 8091:8091
+```
+1. Create a Trillian tree:
+```
+RESPONSE=$(curl \
+--silent \
+--request POST \
+http://localhost:8091/v1beta1/trees \
+--data '{ "tree":{ "tree_state":"ACTIVE", "tree_type":"LOG", "hash_strategy":"RFC6962_SHA256", "signature_algorithm":"ECDSA", "max_root_duration":"0", "hash_algorithm":"SHA256" }, "key_spec":{ "ecdsa_params":{ "curve":"P256" } } }')
```

Comment:
Probably don't want root duration 0 if this tree is going to be used for CT later. Maybe set this to 1 hour.

Reply:
I read then lost this guidance, sorry.

I'll update the post.

I have almost no idea how Trillian works, I'm pretty much resorting to re-engineering stuff that I encounter. So my apologies for my continued ignorance.

When you say "1 hour", what's the unit of `max_root_duration` (seconds? hours?).

Reply:
It's a Duration flag so you include the unit. You should be able to set it to `1h`.

Reply:
Ack. Thanks!